gdeploy_config/volume_usmqe_beta_arbiter_2_plus_1x2.create.conf

```diff
@@ -0,0 +1,30 @@
+# USM QE beta configuration for GlusterFS
+#
+# This gdeploy config file assumes that we have:
+# * 6 hosts machines (GlusterFS storage servers)
+# * 2 block devices on each host (storage server) named /dev/vd{b,c}
+#   plus 1 block device as arbiter named (/dev/vdd)
+
+[hosts]
+
+[peer]
+action=probe
+ignore_peer_errors=no
+
+[backend-setup]
+devices=vdb,vdc,vdd
+vgs=vg_usmqe_beta_arbiter_{1,2,3}
+pools=pool_usmqe_beta_arbiter_{1,2,3}
+lvs=lv_usmqe_beta_arbiter_{1,2,3}
+mountpoints=/mnt/brick_usmqe_beta_arbiter_{1,2,3}
+brick_dirs=/mnt/brick_usmqe_beta_arbiter_1/1,/mnt/brick_usmqe_beta_arbiter_2/2,/mnt/brick_usmqe_beta_arbiter_3/3
+
+[volume]
+volname=volume_usmqe_beta_arbiter_2_plus_1x2
+action=create
+brick_dirs=/mnt/brick_usmqe_beta_arbiter_1/1,/mnt/brick_usmqe_beta_arbiter_2/2,/mnt/brick_usmqe_beta_arbiter_3/3
+transport=tcp
+replica=yes
+replica_count=2
+arbiter_count=1
+force=yes
```

Comment:
I would remove the ``force=yes``, because it might cause unexpected behavior without any notification (e.g. RHBZ1498485).